src/SWP/Bundle/FixturesBundle/Resources/themes/DefaultTheme/views/category.html.twig

```diff
@@ -63,3 +63,11 @@
         <!-- End Content -->
     </div>
 {% endblock %}
+
+{% block template_code %}
+<pre class="js-template-code" style="display:none;">{{ source(_self.getTemplateName())|escape('html') }}</pre>
```

Comment:
why it is added twice in same tpl? consider moving it to base.html.yml as mentioned by me before